inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/interpolate.cpp

```diff
@@ -39,6 +53,7 @@ const std::vector<std::vector<size_t>> targetShapes2x = {
 
 const std::vector<ngraph::op::v4::Interpolate::InterpolateMode> modesWithoutNearest = {
         ngraph::op::v4::Interpolate::InterpolateMode::linear,
+        ngraph::op::v4::Interpolate::InterpolateMode::linear_onnx,
```

Comment:
Did you add all new modes to tests?

Reply:
Well, this PR is for enabling linear_onnx mode. So there are all needed tests.

Reply:
What about Nearest and Cubic where are they tested?

Reply:
Here:
`const std::vector<ngraph::op::v4::Interpolate::InterpolateMode> nearestMode = {`
`        ngraph::op::v4::Interpolate::InterpolateMode::nearest,`
`};`
lines 59-61

Reply:
Cubic mode is not yet supported (not a priority)

Reply:
Ok, I got it. Actually I just was confused by tests structures. Why do we need separate tests parameters vectors for each mode containing one element, instead of single tests parameters vector which would have all modes to test: linear, linear_onnx, nearest and so on..

Not that important anyway, so resolving